packages/@webex/plugin-meetings/src/locus-info/index.ts

```diff
@@ -1377,32 +1377,33 @@ export default class LocusInfo extends EventsScope {
           }
         );
       }
-      if (parsedSelves.updates.localAudioUnmuteRequiredByServer) {
+      if (parsedSelves.updates.isMutedByOthersChanged) {
```

Comment:
diff looks weird, but i basically just switched `isMutedByOthersChanged` to happen before `localAudioUnmuteRequiredByServer`